frontend/src/components/language-menu.tsx

```diff
@@ -19,18 +19,20 @@ const LanguageMenu = (): JSX.Element => {
         window.dispatchEvent(new Event('storage'));
     }, [language]);
 
-    const clicked = () => {
-        if (language === 'no') {
-            setLanguage('en');
-            return;
-        }
-        setLanguage('no');
+    const clicked = (language: Flag) => {
+        setLanguage(language);
     };
-
+    type Flag = 'no' | 'en';
     return (
-        <Button bg="none" fontSize="xxl" onClick={clicked} marginRight="1rem">
-            {language === 'en' ? '🇬🇧' : '🇳🇴'}
-        </Button>
+        <Flex justify={'center'}>
```

Comment:
Bruk `justify="center"`, men også her tror jeg du egt. vil bruke `alignItems`.

```suggestion
        <Flex alignItems="center">
```

Reply:
Må ha begge for at den legger seg riktig med de andre